ui/pages/notifications/notification-components/feature-announcement/feature-announcement.tsx

```diff
@@ -103,26 +110,17 @@ export const components: NotificationComponent<FeatureAnnouncementNotification>
     },
     footer: {
       type: 'footer_feature_announcement',
-      Link: ({ notification }) =>
-        notification.data.link ? (
+      ExtensionLink: ({ notification }) =>
+        notification.data.extensionLink ? (
           <NotificationDetailButton
             variant={ButtonVariant.Primary}
-            text={notification.data.link.linkText}
-            href={notification.data.link.linkUrl}
-            id={notification.id}
-            isExternal={notification.data.link.isExternal}
-            endIconName={false}
-          />
-        ) : null,
-      Action: ({ notification }) =>
-        notification.data.action ? (
-          <NotificationDetailButton
-            variant={ButtonVariant.Secondary}
-            text={notification.data.action.actionText}
-            href={notification.data.action.actionUrl}
+            text={notification.data.extensionLink.extensionLinkText}
+            href={
+              destination + notification.data.extensionLink.extensionLinkRoute
```

Comment:
Instead of this, and instead of putting `extensionLinkRoute` on the controller, and the above logic to create `destination`, you should just be able to use `global.platform.getExtensionURL()`, which refers to https://github.com/MetaMask/metamask-extension/blob/develop/app/scripts/platforms/extension.js#L81 and returns (for example) `chrome-extension://gbaedmjhafpdhbkcbjfebamjmifpgjph/home.html`

Reply:
Thanks @danjm . Actually, `extensionLinkRoute` is used in a different way. `extensionLinkRoute` is a field returned from Contentful. The idea is that in this field, our marketing colleagues can indicate the page a certain announcement should point to. For example, the announcement "We have notifications" could point to `chrome-extension://ehkbeholfhcimjldhdaffdhalaalmdkp/home.html#notifications`.

We have two possibilities then:
1. Our colleagues on Contentful indicate the route "home.html#notifications" in the `extensionLinkRoute` field (which is more intuitive to me, but that's just my opinion).
2. Our colleagues on Contentful indicate the route "#notifications" in the `extensionLinkRoute` field (which might be simpler).

@Prithpal-Sooriya what do you think?

Reply:
Okay, I see. So how about we make this change:
![Screenshot from 2024-05-24 06-34-55](https://github.com/MetaMask/metamask-extension/assets/7499938/cbc83cb9-7d81-43c8-879b-1c60b34847d6)

And then we can replace the above logic to create `destination` with just `global.platform.getExtensionURL(null,null,'')`

Reply:
@danjm thx. I did!

Reply:
🥇